R/pr.R

```diff
@@ -629,14 +632,10 @@ pr_clean <- function(number = NULL,
 # we're in DEFAULT branch of a fork. I wish everyone set up DEFAULT to track the
 # DEFAULT branch in the source repo, but this protects us against sub-optimal
 # setup.
-pr_pull_source_override <- function(tr = NULL, default_branch = NULL) {
-  # naive selection of target repo; calling function should analyse the config
-  tr <- tr %||% target_repo(github_get = FALSE, ask = FALSE)
-
+pr_pull_source_override <- function(tr, default_branch) {
```

Comment:
All callers of `pr_pull_source_override()` supply their arguments and I think we should continue to enforce that.

Reply:
I definitely remember that the only reason I had `NULL` defaults here was for development convenience.